docs/source/api/v4/user_login.rst

```diff
@@ -25,6 +25,7 @@ Authentication of a user using username and password. Traffic Ops will send back
 
 :Auth. Required: No
 :Roles Required: None
+:Permissions Required: None
```

Comment:
Currently we have undocumented special handling for a Role with the exact name "disallowed" that prevents users from logging in. That's because some organizations are not allowed to delete user data (and also to preserve foreign keys on e.g. changelogs and inactive DS Requests), so to prevent old accounts from being used after their owner no longer has need for them, we need to be able to disallow login for some users. We could add an `AUTH` Permission to fill that role, or add a flag to Roles or Users to define the concept. What do you think?

Reply:
Why don't we just have a `locked` column on the user table or something of that sort? This does not seem like an issue we should be tackling with permissions.